DMFT/src/main/java/com/gov/rsi/DMFT/models/Report.java

```diff
@@ -1,48 +1,47 @@
 package com.gov.rsi.DMFT.models;
 
-import org.bson.types.ObjectId;
+import java.time.LocalDateTime;
+
 import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.index.Indexed;
 import org.springframework.data.mongodb.core.mapping.Document;
 
+import lombok.Data;
+
+/**
+ * A Report wraps the DMER JSON document along with status and timestamps to 
+ * implement a queue
+ */
 @Document
+@Data
 public class Report {

```

Comment:
I'd suggesting renaming this to Dmer, or something that relates it to the DMER.

Reply:
Yes, per above comment